kolla/defaults.yml

```diff
@@ -257,6 +257,7 @@ neutron_dev_plugins:
 
 # minimal config for neutron wireguard agent
 neutron_wireguard_hub_endpoint: "{{ kolla_external_vip_address }}"
+create_hub_in_root_netns: "{{ create_hub_in_root_netns }}"
```

Comment:
this needs a default:

normally I would say to default it to `false` so that just deploying the container won't change behavior.
however, as we're the only ones deploying this, lets set it to `true` because it will save some steps.